articles/Why-julia-is-slowly-replacing-python-for-machine-learning-and-data-science/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/why-julia-is-slowly-replacing-python-for-machine-learning-and-data-science/
+title: Why Julia is Slowly Replacing Python for Machine Learning and Data Science
+description: Jupyter Notebook is a standard tool in data science because of its ability to allow users to plot and explore data.
+author: eric-kahuha
+date: 2020-10-T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/why-julia-is-slowly-replacing-python-for-machine-learning-and-data-science/hero.jpg
+    alt: Julia Python Machine Learning example image
+---
+As the demand for data manipulation and scientific computation arose, so did the need for a better data processing language. A group of mathematicians and computer scientists led by Alan Edelman, Viral B. Shah, Jeff Bezanson, and Stefan Karpinski, noted that the then-existing general-purpose languages were not excellent for logical and arithmetic coding.
+<!--more-->
+### What is Julia Programming Language?
+Particularly, there was a lack of a programming language that could perform these tasks at a lightning-fast speed. Thus, they resolved to create a high-performance programming language for machine learning and data science. And that is how [Julia](https://julialang.org/) was introduced. Since its launch in 2012, Julia has become a trendy language among data scientists and mathematicians.
+
+Among the current corporate users of Julia are Aviva and BlackRock. They use this tool to calculate risks and execute time-series analysis, respectively. The Celeste project, which is based on the Julia language set a new scientific record due to its fast speed in cataloguing telescopic data for astronomical objects. The Climate Modeling Alliance, CISCO, QuantEcon, NVIDIA, among others also use Julia. In fact, Julia has become a primary tool in the fields of data science, visualization, machine learning and artificial intelligence.
```

Comment:
Links to the companies and organizations would be helpful for the reader in case they are interested to see what you are talking about.